src/main/app/claims/claimStoreClient.ts

```diff
@@ -32,6 +33,16 @@ export default class ClaimStoreClient {
     })
   }
 
+  static saveOfferForUser (user: User, offer: Offer): Promise<void> {
+    const claim: Claim = user.claim
+    return request.post(`${claimStoreOfferApiUrl}/${claim.id}/defendant/${user.id}`, {
```

Comment:
a claimant or defendant can have an offer, not sure if this API is right or whether the name of the method needs to change to `saveOfferForDefendant`